meta/documents/user_guide/de/settings_guide.md

```diff
@@ -0,0 +1,175 @@
+
+2. Suchen Sie nach Artikeln, die auf Wayfair verkauft werden sollen
+
+3. **Klicken Sie für jeden Artikel** auf den Artikel in den Suchergebnissen und dann auf `Varianten` ![plus button](../../../images/common/button_plus.png)
+
+4. **Für jede der Varianten**:
+    1. Klicken Sie auf die Registerkarte `Einstellungen`
+    2. Wählen Sie im Abschnitt "Barcode" einen Barcode-Typ aus, klicken Sie auf die Schaltfläche `Hinzufügen` und geben Sie den Barcode-Wert in das Feld `1`Code` ein. Der Barcode sollte mit der Wayfair-Lieferanten-Teilenummer übereinstimmen.
+    3. Klicken Sie auf der Variantenstufe auf die Schaltfläche `Speichern` ![save button](../../../images/common/button_save.png) (nicht zu verwechseln mit der Schaltfläche `Speichern` für den Artikel, einige Zeilen darüber).
+
+#### Verwenden der SKU als Elementzuordnungsmethode
+Wenn die "Artikelzuordnungsmethode" auf "SKU" eingestellt ist, sollte jede Artikelvariation in Plentymarkets so eingerichtet werden, dass sie eine Wayfair-spezifische SKU enthält, die die Wayfair-Lieferanten-Teilenummer widerspiegelt, die Wayfair in eingehenden Bestelldaten sendet:
+
+![sku field](../../../images/de/variation_sku_field.png)
+
+1. Gehen Sie auf der Hauptseite von Plentymarkets zu `Artikel` >>` Artikel bearbeiten`
+
+2. Suchen Sie nach Artikeln, die auf Wayfair verkauft werden sollen
+
+3.  **Klicken Sie für jeden Artikel** auf den Artikel in den Suchergebnissen und dann auf `Varianten`
+
+4. **Für jede der Varianten**:
+    1. Klicken Sie auf die Registerkarte `Verfügbarkeit`
+    2. Klicken Sie im Abschnitt `SKU` auf die Schaltfläche `Hinzufügen` ![plus button](../../../images/common/button_plus.png)
+    3. Wählen Sie im Popup-Fenster `Wayfair` für `Herkunft`, geben Sie die Wayfair-Lieferanten-Teilenummer in das Feld `SKU` ein und klicken Sie auf `Hinzufügen`
+    4. Klicken Sie auf "Hinzufügen" ![add button with text](../../../images/de/button_plus_add.png)
+    5. Klicken Sie auf der Variantenebene auf die Schaltfläche `Speichern` ![save button](../../../images/common/button_save.png) (nicht zu verwechseln mit der Schaltfläche `Speichern` für den Artikel, einige Zeilen darüber).
+
+
+### Aufträge importieren ab
+Die optionale Einstellung `Aufträge importieren ab` bestimmt ein Datum, ab dem neue Wayfair-Bestellungen als für den Import in das Plentymarkets-System berechtigt gelten. Diese Einstellung kann die Leistung verbessern, indem verhindert wird, dass das Wayfair-Plugin Bestellungen anfordert, die vor dem angegebenen Datum erstellt wurden. Es kann auch verwendet werden, um zu verhindern, dass das Plentymarkets-System Bestellungen vor einem gewünschten "Go-Live-Datum" annimmt.
+
+### Vollständigen Bestand an Wayfair senden?
+Die optionale Einstellung `Vollständigen Bestand an Wayfair senden?` legt fest, ob Plentymarkets-Artikelvariationen ausdrücklich als für den Verkauf auf Wayfair geeignet deklariert werden müssen oder nicht.

```

Comment:
Just double check if the Plentymarkets UI actually says `Vollständigen Bestand an Wayfair senden?` 
In the English version the optional requirement settings ays "Send all inventory items to Wayfair?" but I don't see this in the screenshot.